.github/workflows/build.yml

```diff
@@ -23,12 +23,12 @@ jobs:
           cache: "maven"
 
       - name: Install Missing Dependencies
-        run: sudo apt-get install -y libv4l-0 libopencv-dev python3-opencv
+        run: sudo apt-get install -y libv4l-0 libopencv-dev python3-opencv libgtk2.0-0
 
       - name: Dependency Test # installs all dependencies
         run: mvn test -Dtest=org.myrobotlab.framework.DependencyTest -q
       - name: Build with Maven # currently cannot test opencv
-        run: mvn clean verify -q
+        run: mvn clean verify -q -Dtest=!org.myrobotlab.opencv.*
```

Comment:
This should be temporary, is this done to speed up the tests or are there too many failures currently?

Reply:
opencv testing on github actions currently fails the dep add might have helped, but the resources are still probably too low - an actual release on Jenkins doesn't filter